Sources/DocCDocumentation/DocCDocumentation.docc/Purchases.md

```diff
@@ -22,7 +22,7 @@ Most features require configuring the SDK before using it.
 - ``Purchases/logHandler``
 
 ### Configuring the SDK
-- ``Purchases/configure(withAPIKey:)``
+- ``Purchases/configure(withAPIKey:storeKitVersion:)``
```

Comment:
honestly I think this kind of defeats the purpose of having these functions in the first place - they're meant to keep configuration as easy as possible for the majority of the customer base. 

Most customers should not have to think about the storekit version, they pay us to do it for them.